VERSIONS.md

```diff
@@ -0,0 +1,24 @@
+# Version Lifecycle
+
+We use a modified version of [Semantic Versioning](https://semver.org/) for all changes. It is strongly encouraged that you pin at least the major version and potentially the minor version to avoid pulling in breaking changes.  Increasing the major version of an SDK indicates that this SDK underwent significant and substantial changes to support new idioms and patterns in the language. Major versions are introduced when public interfaces (e.g. classes, methods, types, etc.), behaviours, or semantics have changed. 
+
+Semantic Versions take the form of `MAJOR.MINOR.PATCH`.
+
+When bugs are fixed in the library in a backwards-compatible way, the PATCH level will be incremented by one. When new features are added to the library in a backwards-compatible way, the PATCH level will be incremented by one. PATCH changes should not break your code and are generally safe for upgrade.
```

Comment:
Actually currently we have standard versioning set, that even small new feature will cause minor version increment. Should we adjust it somehow to fit this description?

Reply:
changed it to the following wording:  "We use a [Semantic Versioning](https://semver.org/) for all changes."

Reply:
` When new features are added to the library in a backwards-compatible way, the PATCH level will be incremented by one.` This sentence is not accurate with our release versioning. When at least one commit with feat: prefix is found since last release - a minor version is increased. My question was: should we change the currently implemented versioning behaviour or do we change the description and specify that ANY new feature will cause minor version increment.

Reply:
fixed. Please take a look